micromamba/etc/profile.d/mamba.sh.in

```diff
@@ -1,19 +1,7 @@
-echo "WARNING: @CMAKE_INSTALL_PREFIX@/etc/profile.d/mamba.sh (the file emitting this warning) is deprecated."
-echo "WARNING: This file will be removed on the next release after 30th September 2025."
-echo "WARNING: Please use 'mamba shell init' to get the correct initialization for your shell."
-
 if [ -z "${MAMBA_ROOT_PREFIX}" ]; then
-    echo "WARNING: The MAMBA_ROOT_PREFIX environment variable is not set."
-    echo "WARNING: This is required for mamba to work correctly as of 2.0."
-    echo "WARNING: "
-    echo "WARNING: For now, we are setting 'MAMBA_ROOT_PREFIX' to '@CMAKE_INSTALL_PREFIX@'."
-    echo "WARNING: "
-    echo "WARNING: Please make sure this is consistent with your installation or alternatively (by order of preference):"
-    echo "WARNING:   - rerun 'mamba shell init' to initialize mamba for your current shell"
-    echo "WARNING:   - manually set 'MAMBA_ROOT_PREFIX' to the root of your installation in your shell profile script."
-    echo "WARNING:   - use the '-r,--root-prefix' CLI option when calling mamba."
-    echo "WARNING: "
-    echo "WARNING: This message originates from @CMAKE_INSTALL_PREFIX@/etc/profile.d/mamba.sh"
+    echo "WARNING: 'MAMBA_ROOT_PREFIX' is not set."
+    echo "WARNING: Please set 'MAMBA_ROOT_PREFIX' to the root of your installation."
+    echo "WARNING: For now continuing with 'MAMBA_ROOT_PREFIX' set to '@CMAKE_INSTALL_PREFIX@'."
```

Comment:
Is there a reason for using single quotes?  `` seems more appropriate?

Reply:
Backquote will execute the content. We got bitten by this in the past, see https://github.com/conda/conda/pull/14455#discussion_r1901556839.